web/pageComponents/shared/SanityImage.tsx

```diff
@@ -41,14 +41,14 @@ const Image = ({
     // Layout fill
     props = {
       fill,
-      style: { ...style, objectFit: 'cover' },
+      style: { objectFit: 'cover' },
     }
   } else {
     // Layout responsive
     props = {
       width,
       height,
-      style: { ...style, display: 'flex', width: '100%', height: 'auto' },
+      style: { display: 'flex', width: '100%', height: 'auto' },
```

Comment:
removing this will remove the propagation of styles everywhere on our codebase

Reply:
there was only one place style was applied using the sanityImage, and it was redundant. Removed in the latest commit. Styling was safely removed without having any implications, this is due to the refactoring of the image component earlier last year.